CodeWars/Tasks/ReturnMaximumHourMinute.cs

```diff
@@ -6,7 +6,7 @@ namespace CodeWars.Tasks
     /// https://www.codewars.com/kata/62a4abf03236976b7acf7781
     /// </summary>
     /// Add string
-    public class ReturnMaximumHourMinute
+    public partial class Tasks
     {
```

Comment:
Let's rename hashMap to be more descriptive.

Reply:
Renamed hashMap to NumberOfElements

Reply:
Also added the "CodeWars.sln" file